.travis.rb

```diff
@@ -36,7 +36,11 @@ def sh!(command)
 sh!('bundle exec codeclimate-test-reporter') if master? && test?
 
 # Running YARD under jruby crashes so skip checking manual under jruby
-sh!('bundle exec rake generate_cops_documentation') unless jruby?
+# Workaround: This task makes a mysterious diff in Ruby 2.0.
+#             So we ignores Ruby 2.0.
+if !jruby? && RUBY_VERSION >= '2.1'
```

Comment:
Guess this can be excluded on rbx as well. I wonder if it shouldn't just be a separate travis task which we ran only on the newest Ruby.